<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="uwidget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50000</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #dddddd</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="action_Source_Images"/>
    <addaction name="action_Moab"/>
    <addaction name="actionFilm"/>
    <addaction name="action_Gpen"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Tools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Source_Images">
   <property name="text">
    <string>&amp;Source Images...</string>
   </property>
  </action>
  <action name="action_Moab">
   <property name="text">
    <string>&amp;Moab</string>
   </property>
  </action>
  <action name="actionFilm">
   <property name="text">
    <string>F&amp;ilm</string>
   </property>
  </action>
  <action name="action_Gpen">
   <property name="text">
    <string>&amp;Gpen</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
